FaaS/OpenFaaS/template/go-clearlinux/Dockerfile

```diff
@@ -0,0 +1,58 @@
+FROM openfaas/classic-watchdog:0.18.1 as watchdog
+
+#FROM golang:1.11-alpine3.10 as builder
+FROM clearlinux/golang as builder
+
+# Allows you to add additional packages via build-arg
+ARG ADDITIONAL_PACKAGE
```

Comment:
@sophia0328 Clear Linux doesn't use this arg, any reason to set it here?

Reply:
check with official GO faas template, it seems this is not necessary. will remove this line.